package.json

```diff
@@ -26,7 +26,7 @@
     "app-devserver": "concurrently --passthrough-arguments --kill-others \"yarn:watch --watchonly {1}\" yarn:app-python-devserver yarn:hashi-dev --",
     "devserver": "concurrently --passthrough-arguments --kill-others \"yarn:watch --watchonly {1}\" yarn:python-devserver yarn:hashi-dev --",
     "devserver-hot": "concurrently --passthrough-arguments --kill-others \"yarn:watch-hot --watchonly {1}\" yarn:python-devserver yarn:hashi-dev --",
-    "devserver-with-kds": "concurrently --passthrough-arguments --kill-others \"yarn:watch --watchonly --kds-path={1}\" yarn:python-devserver yarn:hashi-dev --",
+    "devserver-with-kds": "concurrently --passthrough-arguments --kill-others \"yarn:watch --watchonly --kds --kds-path={1}\" yarn:python-devserver yarn:hashi-dev --",
```

Comment:
One thing that I think might be an issue here is that this seems like it will disallow the use of the baked-in `--watchonly` call which can be passed a comma-separated list of plugins like so IIRC:

`yarn devserver learn,coach` which will restrict the build process to those Kolibri plugins.

The `--kds-path` is swallowing the entirety of what is passes in as the `--watchonly` option will never receive any user input. 

So we cannot call `yarn devserver-with-kds learn,coach` to only build those plugins because the resulting call will be 

`kolibri-tools build dev --file ./build_tools/build_plugins.txt --cache --watchonly --kds-path=learn,coach`

---

Perhaps we can maintain the functionality by changing this line to:

`"devserver-with-kds": "concurrently --passthrough-arguments --kill-others \"yarn:watch --watchonly={2} --kds --kds-path={1}\" yarn:python-devserver yarn:hashi-dev --"`

(note the `={2}` added after `--watchonly`)

In this case, we can call: `yarn devserver-with-kds ../path/to/kds learn,coach` which would result in this call:

`kolibri-tools build dev --file ./build_tools/build_plugins.txt --cache --watchonly=learn,coach --kds-path=../path/to/kds` 

And this should fix the issue here.